From b6134972d45f382eca10f4e8c75ab1c842d86fd8 Mon Sep 17 00:00:00 2001
From: Steve Kondik <steve@cyngn.com>
Date: Mon, 23 Nov 2015 14:06:09 -0800
Subject: [PATCH 1/3] dataservices: Don't pollute the tree

 * Nexus devices have a copy of this in their device repo already.
   Exclude it unless we're doing a QC-style build.

Change-Id: I2a4d3d9656be28b614e3ee59514b54801c833c3e

dataservices: You say BOARD, I say TARGET

 * Fix derp.

Change-Id: I038f4e9618b1ff9b128f73c9f15cf5b71c01f45c

Makefiles: Use project pathmap to determine build eligibility

Change-Id: Ie14dbc002bd06fdbfd85d77b8d0f892d286d889b
---
 datatop/Android.mk  | 3 ++-
 rmnetctl/Android.mk | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/datatop/Android.mk b/datatop/Android.mk
index 8338432..e1c3023 100644
--- a/datatop/Android.mk
+++ b/datatop/Android.mk
@@ -1,2 +1,3 @@
+ifeq ($(call my-dir),$(call project-path-for,qcom-dataservices))
 include $(call all-subdir-makefiles)
-
+endif
diff --git a/rmnetctl/Android.mk b/rmnetctl/Android.mk
index 8338432..e1c3023 100644
--- a/rmnetctl/Android.mk
+++ b/rmnetctl/Android.mk
@@ -1,2 +1,3 @@
+ifeq ($(call my-dir),$(call project-path-for,qcom-dataservices))
 include $(call all-subdir-makefiles)
-
+endif
-- 
2.34.0

